install.sh

```diff
@@ -32,4 +27,11 @@ rm -rf node_modules
 npm install
 cd ../..
```

Comment:
Above installing analytics-vendors is not needed as it is already installed in the `browser-extension/install.sh`

Reply:
yeah, we can choose to keep the analytics-vendors `npm i` in either here or in browser-extention/install.sh

I have removed it from browser-extention/install.sh, do you want it other way round?